desc/objectives/utils.py

```diff
@@ -224,8 +224,9 @@ def factorize_linear_constraints(objective, constraint, x_scale="auto"):  # noqa
         # else check with tighter tols and throw an error, these tolerances
         # could be tripped due to just numerical roundoff or poor scaling between
         # constraints, so don't want to error out but we do want to warn the user.
-        atol = 3e-14
-        rtol = 3e-14
+        size = max(A.shape)
+        atol = 1e-15 * size
```

Comment:
does this help a lot? if so we should put it in a separate PR instead of this on

Reply:
I don't remember but we discussed this in one of the dev meetings. I am down for putting in a new PR.